package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "full-icu",
-  "version": "1.3.5-0",
+  "version": "1.4.0-0",
   "description": "install 'full-icu' data for your current node",
   "scripts": {
-    "lint": "standard",
-    "postinstall": "node postinstall.js"
+    "lint": "standard && eslint *.js test/*.js",
```

Comment:
Think you can skip the call to `standard` since you're using the ESlint plug-in now